builtin/game/falling.lua

```diff
@@ -146,13 +146,28 @@ end
 
 local function drop_attached_node(p)
 	local n = core.get_node(p)
+	local node_meta, meta_fields = core.get_meta(p), nil
+	if node_meta then meta_fields = node_meta:to_table().fields end
```

Comment:
`local meta_fields = node_meta and node_meta:to_table().fields`

Reply:
Looks good.